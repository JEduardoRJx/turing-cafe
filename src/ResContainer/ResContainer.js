import React from 'react'
import './ResContainer.scss'

const ResContainer = () => {
  return (
    <section>
    Reservation Container
    </section>
  )
}

export default ResContainer;