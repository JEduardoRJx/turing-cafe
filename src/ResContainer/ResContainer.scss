section {
  display: flex;
  padding: 30px;
  justify-content: space-around;
  width: 100%;
  height: 100%;
}