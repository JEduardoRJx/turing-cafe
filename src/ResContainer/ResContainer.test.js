import React from 'react';
import ResContainer from './ResContainer';
import { shallow } from 'enzyme'

describe('ResContainer', () => {
  let wrapper, mockGuests;

  beforeEach(() => {
    mockGuests = []
    wrapper = shallow(<ResContainer guests={mockGuests}/>)
  });

  it('should create a snapshot of ResContainer', () => {
    expect(wrapper).toMatchSnapshot();
  });

})