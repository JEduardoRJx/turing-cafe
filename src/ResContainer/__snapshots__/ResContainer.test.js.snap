// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ResContainer should create a snapshot of ResContainer 1`] = `<section />`;
